cranelift/2025/cranelift-11-05.md

```diff
@@ -17,5 +17,36 @@
 
 ### Attendees
 
+- fitzgen
+- bjorn3
+- Amanieu
+- uweigand
+- alexcrichton
+- cfallin
+
 ### Notes
 
+- Updates
+  - alexcrichton: no updates
+  - uweigand: no updates
+  - Amanieu: no updates
+    - abrown mentioned when back from sabbatical may have time to review
+      regalloc3
+  - cfallin:
+    - update on plan for debugging
+      (https://github.com/bytecodealliance/wasmtime/issues/11964)
+    - drafted a blog post on exception handling; posting soon
+  - fitzgen: no updates, Wasm CG last week (reviewed exception handling post;
+    thanks!)
+
+- alexcrichton: debugging and threads
+  - store will own a private copy; all's fine; stop-everything behavior is on
+    the host if other stores/instances also have access to a shared memory
+
+- Cranelift as a production rustc backend
+  - main issue is performance with unwinding enabled
+  - (discussion about performance implications of more blocks)
+  - bjorn3: 7% compile-perf impact with unwinding
```

Comment:
The 7% figure was for compiling with MIR and Cranelift optimizations. I don't recall the exact compile perf hit for unwinding, but I believe it was a bit less, but still non-trivial.